app/components/satis/dropdown/component.scss

```diff
@@ -5,6 +5,16 @@
   }
 
   input {
-    @apply dark:text-gray-300
+    @apply dark:text-gray-300;
+    
   }
 }
+
+.warning
```

Comment:
Could you move this inside the satis-dropdown in a some way?

Reply:
moved